examples/gallery/line/linefronts.py

```diff
@@ -0,0 +1,84 @@
+"""
+Line fronts
+-----------
+
+Using the :meth:`pygmt.Figure.plot` method you can draw a so-called
+*front* which allows to plot specific symbols distributed along a line
+or curve. Typical use cases are weather fronts, fault lines,
+subduction zones, and more.
+
+A front can be drawn by passing **f**\[±]\ *gap*\[/*size*] to the ``style`` parameter where *gap*
+defines the distance gap between the symbols and *size* the symbol size.
+If *gap* is negative, it is interpreted to mean the number of symbols
+along the front instead. If *gap* has a leading + then we use the value
+exactly as given [Default will start and end each line with a symbol,
+hence the *gap* is adjusted to fit]. If *size* is missing it is set to 30%
+of the *gap*, except when *gap* is negative and *size* is thus required.
```

Comment:
Do we want to have a list of optional arguments here as https://docs.generic-mapping-tools.org/latest/plot.html#s so that the reader can know the full list while they can read more details of each one?: `[+l|+r][+b+c+f+s+t][+ooffset][+p[pen]].`

```suggestion
of the *gap*, except when *gap* is negative and *size* is thus required.
Append more elements to plot different fronts
[**+l**\|\ **+r**\ ][**+b**\ **+c**\ **+f**\ **+s**\ **+t**\][**+o**\ *offset*][**+p**\[*pen*]].
```
![Screenshot from 2021-03-11 16-47-15](https://user-images.githubusercontent.com/50591376/110760234-76313980-8289-11eb-9f05-c3f53188b155.png)

Reply:
In principle that's a good idea, however, I'm not sure if we should add it here since the indiviudal options are described afterwards. What do the others think @GenericMappingTools/python-contributors ?

Reply:
Perhaps not. I don't think we should explain the full syntax of one parameter in a gallery example.

Reply:
Ok, than we leave it as is.